### PR TITLE
Document GOV.UK Notify Emergency Alerts

### PR DESCRIPTION
**Pull request by richardTowers**
Including how to rotate API keys in Fastly.

screenshot of the rotate API keys page
screenshot of the what is this page

### CODE REVIEW

File: source/manual/notify-emergency-alerts.html.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+parent: "/manual.html"
+section: Notify Emergency Alerts
+important: true
+---
+
+Emergency alerts are a new way for the emergency services to warn people when lives are in danger.
+
+In an emergency, mobile phones and tablets in the affected area will receive an alert. The alert will tell people what they need to do to stay safe.
+
+This service is being developed by the GOV.UK Notify team and mobile phone networks.
```
Comment (quis) [inline source/manual/notify-emergency-alerts.html.md:15]:
```suggestion
The Cabinet Office and the Department for Digital, Culture, Media & Sport are developing a ‘cell broadcast’ alert system to enable people whose lives are at risk in an emergency to be rapidly contacted via their mobile phone. They are currently at the testing phase and subject to successful progress, hope to launch the service soon.
```

**Review by thomasleese — APPROVED** [review]

File: source/manual/notify-emergency-alerts.html.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+owner_slack: "#govuk-notify-emergency-alerts"
+title: What is GOV.UK Notify Emergency Alerts?
+layout: manual_layout
+type: learn
+parent: "/manual.html"
+section: Notify Emergency Alerts
+important: true
```
Comment (ChrisBAshton) [inline source/manual/notify-emergency-alerts.html.md:8]: What does `important: true` do? I've scanned govuk-developer-docs, tech-docs-gem and tech-docs-template and couldn't see any reference to functionality.

Comment (thomasleese) [reply source/manual/notify-emergency-alerts.html.md:8]: It controls what gets shown in this [`Common tasks`](https://docs.publishing.service.gov.uk/manual.html#common-tasks) section I believe:

https://github.com/alphagov/govuk-developer-docs/blob/ec5e0afaa3df4daefc2d1cebc8a34b4872d83486/app/manual.rb#L10-L16

**Comment by thomasleese** [comment]
Can we merge this in now that https://www.gov.uk/alerts is live?

**Comment by richardTowers** [comment]
Yep, I intended to merge it earlier, but holidays got in the way 🍹 

Not sure whether it should have `important: true` set, I think I just copied that frontmatter from another page without thinking. Let's merge it and see how it looks - if it's annoying we can always remove it in a future PR.

File: source/manual/notify-emergency-alerts.html.md (hunk 2 above)
Comment (ChrisBAshton) [reply source/manual/notify-emergency-alerts.html.md:8]: Ah yes - https://github.com/alphagov/govuk-developer-docs/blob/ec5e0afaa3df4daefc2d1cebc8a34b4872d83486/app/manual.rb#L45

**Pull request merged**